config/checkstyle_checks.xml

```diff
@@ -266,7 +266,7 @@
     <module name="MissingOverride"/>
     <module name="PackageAnnotation"/>
     <module name="SuppressWarnings">
-      <property name="format" value="^((?!unchecked|deprecation|rawtypes|resource).)*$"/>
+      <property name="format" value="^((?!unchecked|deprecation|rawtypes|resource|CheckReturnValue).)*$"/>
```

Comment:
We need a permanent solution for it, there are a lot of Bug Patterns (Checks) in error-prone, and we cannot allow all, some names might even overlap with IDEA or PMD.

Reply:
One more reason to execute it not by maven plugin and keep suppression of stuff in separate file as we did for pitest.

Reply:
How are we exactly planning to do that? Error prone does not support individual suppressions by command line too, am I missing something here?

Reply:
Same as in pitest: get report from CLI, compare with known output, on any diff fail build

Reply:
Still, we can skip command line, just set [`<failOnError>`](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#failOnError) to false, create a separate profile, execute it, get the results, our custom script will parse the results and pass or fail the build.